Assets/Scripts/UI/InGameUI/MenuLeft/MenuLeft.cs

```diff
@@ -45,11 +41,11 @@ public void OpenMenu(string menuName)
         }
     }
 
-    public void CloseMenu()
+    public void CloseCurrentlyMenu()
```

Comment:
The name "CloseOpenedMenu" or "CloseCurrentMenu" would make this clearer as to what is happening.